Source/WarpX.H

```diff
@@ -1035,17 +1035,43 @@ private:
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Efield_avg_fp;
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Bfield_avg_fp;
 
-    //EB grid info
-    amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Venl;
+    // EB: Lengths of the mesh edges
```

Comment:
This makes it a Doxygen comment, so it will show up here: https://warpx.readthedocs.io/en/latest/_static/doxyhtml/class_warp_x.html#pri-attribs as well as editors/IDEs that parse Doxygen (many do :))
```suggestion
    //! EB: Lengths of the mesh edges
```

More info on Doxygen style comments here: https://warpx.readthedocs.io/en/latest/developers/documentation.html